storm-client/src/jvm/org/apache/storm/trident/operation/DefaultResourceDeclarer.java

```diff
@@ -29,8 +33,8 @@
  */
 public class DefaultResourceDeclarer<T extends DefaultResourceDeclarer> implements ResourceDeclarer<T>, ITridentResource {
 
-    private Map<String, Number> resources = new HashMap<>();
-    private static Map<String, Object> conf = Utils.readStormConfig();
+    private final transient Map<String, Number> resources = new HashMap<>();
```

Comment:
In trident Node inherits from DefaultResourceDeclarer, and Node is serialized out (repeatedly for different bolts and spouts in the topology).  These are never used after the topology is created so this reduces the serialized size of a trident topology.

I'll add a comment in the code to make it clean why.